crates/stages/src/pipeline/ctrl.rs

```diff
@@ -10,14 +12,20 @@ pub enum ControlFlow {
         /// The block that caused the unwind.
         bad_block: SealedHeader,
     },
-    /// The pipeline is allowed to continue executing stages.
+    /// The pipeline made progress.
     Continue {
-        /// Block number reached by the stage.
+        /// Block number reached by the last stage in the pipeline.
```

Comment:
It's not the last stage, this enum is also used in the pipeline itself per stage. The block number corresponds to the block number reached by the stage the control flow event is for

Reply:
yes